### ISSUE TITLE
Error on Score Screen after end of the first level

### ISSUE THREAD

**Issue by anastasia-horbachevska** [Cytoid/Cytoid#92]
### Description:
When you start the game for the first time, prohibit the permission for file recording, and restarting the game and allowing  the permission for  file recording, after player passed the first level (Easy LV.1), when he click the "Back" button, an error appears below the screen.

### Precondition:
Phone on Android 10+, without the game.

### Steps to reproduce:
1.  Install the game from Play Store
2. Open the game.
3. Decline the permission for file recording.
4. Close the game.
5. Open the game.
6. Allow the permission for file recording.
6. Click on "Start" button.
7. Pass the level.
8. When "Score Screen" appears, click on "Back" button.

### Actual result:
Player got Error and can't back to the "Menu Screen" until he restart the game.
![photo_2020-01-21_02-07-42](https://user-images.githubusercontent.com/60118831/72764976-4fcebb80-3bfb-11ea-9b29-6378cec695c9.jpg)

```Error
AndroidJavaException: android.os.FileUriExposedException: file:///storage/emulated/0/Android/data/me.tigerhix.cytoid/files/ScreenshotVSSHARE.png exposed beyond app through ClipData.Item.getUri()

android.os.FileUriExposedException: file:///storage/emulated/0/Android/data/me.tigerhix.cytoid/files/ScreenshotVSSHARE.png exposed beyond app through ClipData.Item.getUri()
android.os.StrictMode.onFileUriExposed(StrictMode.java:1978)
android.net.Uri.checkFileUriExposed(Uri.java:2371)
android.content.ClipData.prepareToLeaveProcess(ClipData.java:963)
android.content.Intent.prepareToLeaveProcess(Intent.java:10252)
android.content.Intent.prepareToLeaveProcess(Intent.java:10258)
android.content.Intent.prepareToLeaveProcess(Intent.java:10237)
android.app.Instrumentation.execStartActivity(Instrumentation.java:1669)
android.app.Activity.startActivityForResult(Activity.java:4651)
android.app.Activity.startActivityForResult(Activity.java:4609)
android.app.Activity.startActivity(Activity.java:4970)
android.app.Activity.startActivity(Activity.java:4938)
com.unity3d.player.UnityPlayer.nativeRender(Native Method)
com.unity3d.player.UnityPlayer.c(Unknown Source:0)
com.unity3d.player.UnityPlayer$e$2.queueIdle(Unknown Source:72)
android.os.MessageQueue.next(MessageQueue.java:395)
android.os.Looper.loop(Looper.java:165)
com.unity3d.player.UnityPlayer$e.run(Unknown Source:32)
UnityEngine.AndroidJNISafe.CheckException ()
UnityEngine.AndroidJNISafe.CallVoidMethod (IntPtr obj, IntPtr methodID, UnityEngine.jvalue[] args)
UnityEngine.AndroidJavaObject._Call (System.String methodName, System.Object[] args)
UnityEngine.AndroidJavaObject.Call (System.String methodName, System.Object[] args)
AppAdvisory.SharingSystem.VSSHARE.OpenShareImageMobileNativeDialog (System.String shareText)
AppAdvisory.SharingSystem.VSSHARE.ShareScreenshot (System.String text, ShareType shareType)
AppAdvisory.SharingSystem.VSSHARE.DOShareScreenshot (System.String text, ShareType shareType)
GameResultController.ShareScreenshot (UnityEngine.Texture2D tex)
AppAdvisory.SharingSystem.VSSHARE+<getScreenshot>c__Iterator5.<>m__0 ()
AppAdvisory.SharingSystem.VSSHARE+<DoFlash>c__AnonStorey6.<>m__1 ()
AppAdvisory.SharingSystem.VSSHARE+<DOFade>c__Iterator4.MoveNext ()
UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress)
```
### Expected result:
Player will see "Menu Screen", and not will got Error in the bottom of the screen.

### Additional info:
Tested on devices: Mi Mix 3, Mi Note 7
Game version: 1.5.2

**Comment by yevhen-kalyna** [comment]
@TigerHix Hi, I got the same issue.

**Comment by TigerHix** [comment]
Didn't expect a detailed issue like this, great work! This is fixed in 2.0 (develop branch) which would hopefully be released around mid-March. Thanks again!
